ros/angel_msgs/CMakeLists.txt

```diff
@@ -34,11 +34,13 @@ set( message_files
   msg/HeadsetAudioData.msg
   msg/HeadsetPoseData.msg
   msg/InterpretedAudioUserIntent.msg
+  msg/InterpretedAudioUserEmotion.msg
   msg/ObjectDetection2d.msg
   msg/ObjectDetection2dSet.msg
   msg/ObjectDetection3dSet.msg
   msg/SpatialMesh.msg
   msg/SystemCommands.msg
+  msg/SystemTextResponse.msg
```

Comment:
Should this one be removed here until #297?

Reply:
Fixed. Let's see if that addresses the build error.